index.html

```diff
@@ -0,0 +1,146 @@
+                    <div class="col-sm mb-2">
+                        <button id="button-3" class="btn btn-block btn-primary backspace" value="clear"><i
+                                class="fas fa-backspace h3"></i></button>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!--Tip amount input -->
+        <div class="row">
+            <div class="col-sm text-center">
+                <label for="tipPercentage" class="h4">Tip</label>
+                <div class="input-group mb-3">
+                    <input type="number" min="0" max="100" id="tipPercentage" value="18"
+                        class="form-control form-control-lg" aria-label="Tip percantage between 0 and 100" />
+                    <div class="input-group-append">
+                        <span class="input-group-text">%</span>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!--Split between options -->
+        <div class="row" id="split-area">
+            <div class="col-sm">
+                <label for="splitNumber" class="h4">Split</label>
+                <div class="input-group mb-3">
+                    <button type="button" id="btn-decrease" class="btn btn-primary" aria-label="Decrease"><i
+                            class="fas fa-minus"></i></button>
+                    <span id="splitNumber" class="h2 mx-2">1</span>
+                    <button type="button" id="btn-increase" class="btn btn-primary" aria-label="Increase"><i
+                            class="fas fa-plus"></i></button>
+                </div>
+            </div>
+            <div class="col-sm">
+                <label for="splitTotal" class="h4">Split Total</label>
+                <div class="input-group mb-3">
+                    <span id="splitTotal" class="h2">$ 0.00</span>
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col mb-2">
+                <button id="btn-calculate" class="btn btn-success btn-lg btn-block">
+                    Calculate
+                </button>
+            </div>
+        </div>
+    </div>
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
+        integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+        integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+        crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
+        integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
+        crossorigin="anonymous"></script>
+    <script src="https://kit.fontawesome.com/e67b748ce1.js" crossorigin="anonymous"></script>
+    <script src="script.js" type="text/javascript">
+    </script>
```

Comment:
From what I'm reading here, you're using jquery, bootstrap, fontawesome, and your own script.js. What is jsdelivr and how is it being used?

Reply:
This is the live [link](https://muzhaqi16.github.io/2020-04-01-Interactive-Calculator/) . jsdeliver seems to be the boileplate from bootsrap I never ended up using it.